src/world_interface/real_world_interface.cpp

```diff
@@ -75,9 +83,18 @@ void initMotors() {
 		can::motor::initMotor(serial);
 		can::motor::initEncoder(serial, invEnc, true, ppjr, TELEM_PERIOD);
 		can::motor::setMotorPIDConstants(serial, pid.kP, pid.kI, pid.kD);
+
+		// initialize motor objects and add them to map
+		addMotorMapping(motor, true);
 	}
 
 	can::motor::initMotor(motorSerialIDMap.at(motorid_t::hand));
+	addMotorMapping(motorid_t::hand, true);
+}
+
+void addMotorMapping(motorid_t motor, bool hasPosSensor) {
+	std::shared_ptr<robot::base_motor> ptr(new can_motor(motor, hasPosSensor));
```

Comment:
We prefer to use `make_shared` instead of the `new` keyword, since that allows us to treat all usages of `new` as a code smell. So, this should be:
```c++
std::shared_ptr<robot::base_motor> ptr = std::make_shared<can_motor>(motor, hasPosSensor);
```